docs/blog/posts/2024/week33.md

```diff
@@ -0,0 +1,26 @@
+---
+title: '@warp-ds v2 - Stable releases'
+date: 2024-08-15
+---
+
+We now have stable releases out for web with DBA changes.
+
+---
+
+# How to Update Your Web Apps for DBA with stable releases
+
+### Packages to Install from NPM
+
+| Package Name        | Version        |
+|---------------------|----------------|
+| `@warp-ds/css`      | `2.0.0`        |
+| `@warp-ds/uno`      | `2.0.0`        |
+| `@warp-ds/react`    | `2.0.0`        |
+| `@warp-ds/vue`      | `2.1.0`        |
+| `@warp-ds/elements` | `2.0.0`        |
```

Comment:
Should I make a release of 2.0.1 with the a11y fix of Modal so we can suggest the newest version of elements package here?

Reply:
Hmm yeah that's a good idea!